CHANGELOG.md

```diff
@@ -3,6 +3,153 @@
+- [DataGrid] Rename `linkOperators` to `logicOperators` (#7310) @cherniavskii
+- [DataGrid] Rework column headers and virtual scroller positioning (#7001) @cherniavskii
+- [DataGrid] Stop exporting editing selector (#7456) @m4theushw
+- [DataGrid] Store the outlined cell in the state (#7111) @m4theushw
+- [DataGrid] Update `onColumnOrderChange` behavior to match `onRowsOrderChange` (#7385) @DanailH
+- [DataGridPremium] Fix Excel export not working with date strings (#7396) @cherniavskii
+
+### `@mui/x-date-pickers@6.0.0-alpha.15` / `@mui/x-date-pickers-pro@6.0.0-alpha.15`
+
+### Breaking changes
+
+- [pickers] Stop using the `WrapperVariantContext` in `MonthCalendar` and `YearCalendar` (#7382) @flaviendelangle
+
+  The `modeMobile` and `modeDesktop` classes have been removed from the `PickersMonth` and `PickersYear` internal components.
+
+  If you were using those classes on responsive components,
+  you can import `DEFAULT_DESKTOP_MODE_MEDIA_QUERY` from `@mui/x-date-pickers` or `@mui/x-date-pickers-pro` (or use your custom media query if any):
+
+  ```diff
+  <GlobalStyles
+    styles={{
+  -    [`.${pickersYearClasses.modeDesktop}`]: {
+  -      backgroundColor: 'red'
+  -    }
+  +    [DEFAULT_DESKTOP_MODE_MEDIA_QUERY]: {
+  +      [`.${pickersYearClasses.root}`]: {
+  +        backgroundColor: 'red'
+  +      }
+  +    }
+  -    [`.${pickersYearClasses.modeMobile}`]: {
+  -      backgroundColor: 'red'
+  -    }
+  +    [DEFAULT_DESKTOP_MODE_MEDIA_QUERY.replace('@media', '@media not')]: {
+  +      [`.${pickersYearClasses.root}`]: {
+  +        backgroundColor: 'red'
+  +      }
+  +    }
+    }}
+  />
+  ```
+
+  Works exactly the same way for `PickersMonth`.
+
+- [pickers] Refactor `shouldDisableTime` (#7299) @LukasTy
+
+  `shouldDisableTime` prop signature has been changed. Either rename the prop usage to `shouldDisableClock` or refactor usage.
+
+  ```diff
+  <DateTimePicker
+  -  shouldDisableTime={(timeValue, view) => view === 'hours' && timeValue < 12}
+  +  shouldDisableClock={(timeValue, view) => view === 'hours' && timeValue < 12}
+  />
+
+  <DateTimePicker
+  -  shouldDisableTime={(timeValue, view) => view === 'hours' && timeValue < 12}
+  +  shouldDisableTime={(time, view) => view === 'hours' && value.hour() < 12}
+  />
+  ```
+
+#### Changes
+
+- [fields] Fix Android editing (#7444) @flaviendelangle
+- [pickers] Add Belarusian (be-BY) locale (#7395) @volhalink
+- [pickers] Hide am/pm controls when their is no hour view (#7380) @flaviendelangle
+- [pickers] Hide the tabs by default on `DesktopNextDateTimePicker` (#7503) @flaviendelangle
+- [pickers] Refactor `shouldDisableTime` (#7299) @LukasTy
+- [pickers] Remove `WrapperVariantContext` from `DateTimePickerTabs` (#7374) @LukasTy
+- [pickers] Stop using the `WrapperVariantContext` in `MonthCalendar` and `YearCalendar` (#7382) @flaviendelangle
+- [pickers] Support components and slots for new pickers (#7390) @alexfauquette
```

Comment:
```suggestion
- [pickers] Support `components` and `slots` for new pickers (#7390) @alexfauquette
```

I added a PR #7528 I don't think it need to be added to the changelog since it's a quick fix of #7390